catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-schema-webapp/pom.xml

```diff
@@ -23,15 +23,6 @@
     <name>DDF :: Spatial :: WFS :: 1.0.0 :: Schema WebApp</name>
     <packaging>bundle</packaging>
 
-    <dependencies>
-        <dependency>
```

Comment:
❓ If we are going to import packages from jetty-servlets bundle, wouldn't we still want jetty-servlets as a dependency?

Reply:
Before these changes, I was unable to navigate to https://localhost:8993/ogc/gml/2.1.2/feature.xsd
and make sure it shows the feature.xsd file. I'm not sure if this is important, but this was what I used to test the changes that were added in the original PR.